docs/standard/serialization/snippets/binaryformatter-event-source/csharp/binaryformatter-event-source.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I think I will revert this to .NET 6 and add the project to the `dotnet-versionsweeper.json`. 
BinaryFormatter serialization APIs produce compiler errors [starting in .NET 7](https://learn.microsoft.com/en-us/dotnet/core/compatibility/serialization/7.0/binaryformatter-apis-produce-errors) so the sample either needs to stay on 6 or be removed.